manifests/p/PatrickSiegler/PrismatikUnofficial/5.11.2.31/PatrickSiegler.PrismatikUnofficial.installer.yaml

```diff
@@ -3,6 +3,10 @@
 
 PackageIdentifier: PatrickSiegler.PrismatikUnofficial
 PackageVersion: 5.11.2.31
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
```

Comment:
```suggestion
```
Oops, I over-looked this.

This isn't needed here because `InstallerType` is already defined in https://github.com/microsoft/winget-pkgs/pull/60085/files#diff-c606db9d98fbdfd67ad2e5ea839c3bb565cb499133d8246ce07d9afde00f2df1R12.

Inno = EXE that uses the default InstallerSwitches noted in the winget-cli source code.